qiskit/extensions/quantum_initializer/initializer.py

```diff
@@ -89,9 +90,12 @@ def __init__(self, params, num_qubits=None):
             if num_qubits == 0 or not num_qubits.is_integer():
                 raise QiskitError("Desired statevector length not a positive power of 2.")
 
-            # Check if probabilities (amplitudes squared) sum to 1
-            if not math.isclose(sum(np.absolute(params) ** 2), 1.0, abs_tol=_EPS):
-                raise QiskitError("Sum of amplitudes-squared does not equal one.")
+            if normalize is True:
+                params = self._normalize(params)
+            elif normalize is False:
```

Comment:
Elif is not necessary since this part will only be reached if  `normalize` is `False` anyways
```suggestion
            else:
```

Reply:
Hi Cryoris. `elif` is necessary here because if normalize is not True, it can be anything besides False e.g. normalize=None, but I don't want to allow this.

`test = None
if test:
    print('foo')
else:
    print('blah')`

Prints "blah"

Reply:
We expect users to respect the type hints, so we don't have to cover any cases where `normalize` is not a boolean (since that's what the type hint says) 🙂  

If we did cover any type for any argument our code base would need a lot more typechecks than it has right now! 😄

Reply:
Ah ok I see, thanks for clearing that up for me. Appreciate it.